File: Javascript/ex1001/ex1001.js
/*format of iuput

var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
    
var a = parseInt(lines.shift());
var b = parseInt(lines.shift());
   
console.log('X = ' + (a+b));


*/
var input = require('fs').readFileSync('Javascript/ex1001/olar', 'utf8');
var lines = input.split('\n');
/*format of iuput*/
var NumA = parseInt(lines.shift());
var NumB = parseInt(lines.shift());

console.log('X = ' + (NumA + NumB));

File: GO/ex1001.go
/*
package main

import (
    "fmt"
)

func main() {
    var a,b int
    var x int

    fmt.Scanf("%d", &a)
    fmt.Scanf("%d", &b)
    x = a + b

    fmt.Printf("X = %d\n", x)
}
*/
package main

import (
	"fmt"
)

func main() {
	var NumA, NumB, Sum int
	fmt.Scanf("%d", &NumA)
	fmt.Scanf("%d", &NumB)

	Sum = NumA + NumB
	fmt.Printf("X = %d\n", Sum)
}
